agent/containermetadata/mocks/containermetadata_mocks.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	go_dockerclient "github.com/fsouza/go-dockerclient"
 	gomock "github.com/golang/mock/gomock"
+	"github.com/aws/amazon-ecs-agent/agent/api"
```

Comment:
I think the convention is to order import statements as: 1) native go packages, 2) packages from the same repo, 3) other packages, with an empty line between each group. So this new line should be in 21?